src/progressbar/progressbar.js

```diff
@@ -23,6 +26,10 @@ angular.module('ui.bootstrap.progressbar', [])
             bar.percent = +(100 * value / $scope.max).toFixed(2);
         });
 
+        $scope.$watch('max', function( max ) {
```

Comment:
Should only add watcher in an if block when `if ($attrs.max) {` since otherwise the max will never change.